attributes/default.rb

```diff
@@ -13,8 +12,9 @@
 #
 # Logstash
 #
-default['logstash']['version']                               = "7.2.0"
-default['logstash']['url']                                   = "#{node['download_url']}/logstash-oss-#{node['logstash']['version']}.tar.gz"
+default['logstash']['version']                               = "7.13.2"
+default['logstash']['url']                                   = "#{node['download_url']}/opensearch/logstash-oss-with-opensearch-output-plugin-#{node['logstash']['version']}-linux-x64.tar.gz"
+#default['logstash']['sha512']                                = "de5c7ee0d1296787032d91733bb18d6cb9669e8887e683930f9d9c285b28e582b4b4aaf2e8e2365283496e71d00baec8dd109f532170f0e0cc88d35497f79424"
```

Comment:
remove sha512 if not used

Reply:
I want to leave it there as a reminder that some day we should validate the file hashes...